wire-compiler/src/main/java/com/squareup/wire/schema/Target.kt

```diff
@@ -372,7 +372,12 @@ data class CustomTarget(
   }
 
   override fun newHandler(): SchemaHandler {
-    return schemaHandlerFactory.create()
+    return schemaHandlerFactory.create(
+      includes = includes,
+      excludes = excludes,
+      exclusive = exclusive,
+      outDirectory = outDirectory,
```

Comment:
I wonder if I need to use reflexion here, check if the method exist before executing it.